packages/core/src/buffer.ts

```diff
@@ -38,3 +48,13 @@ ds.Buffer.prototype.setBit = function setBit(bitindex: number, on: boolean) {
     // save
     this[i] = byte
 }
+
+ds.Buffer.prototype.readUInt16LE = function readUInt16LE(offset: number) {
+    if (offset < 0 || offset + 2 > this.length) return 0
```

Comment:
we do have `Buffer.get("u16", offset)` (though these are always LE)

Reply:
Ha! I missed those.